src/Stryker.Core/Stryker.Core/TestRunners/VsTest/VsTestContextInformation.cs

```diff
@@ -93,19 +93,22 @@ public void Dispose()
     }
 
     /// <summary>
-    ///     Starts a new VsTest instance and returns a wrapper to control it.
+    /// Starts a new VsTest instance and returns a wrapper to control it.
     /// </summary>
     /// <param name="runnerId">Name of the instance to create (used in log files)</param>
```

Comment:
Add controlVariable to the xml docs

Reply:
Or rename to something descriptive, like `mutantId`

Reply:
added to the doc.